scss/lessonCard.scss

```diff
@@ -0,0 +1,43 @@
+* {
+  color: #292929;
+}
+.card {
+  max-width: 308px;
```

Comment:
doesn't this change ALL card classes? including the default bootstrap ones? Is that really what we want to do? I don't think this should be here.